### DISCUSSION TITLE
Welcome to RedNotebook Discussions!

### DISCUSSION THREAD

**Discussion by jendrikseipp** [jendrikseipp/rednotebook#547] [General]
## 👋 Welcome!
We use Discussions for asking questions, sharing ideas and posting feedback about RedNotebook.

**Comment by tuxedotje** [comment]
in rednotebook Fedora 34 start in terminal :  <08:04:40,079 INFO     Last backup was made -2914143 days ago>
That is a very long time ago...  (about 7000 years)

**Reply by jendrikseipp** [reply]
Hehe, true that :-) Probably we should handle the case that no backup has been made separately...

**Comment by pada-jb** [comment]
Just upgraded to Linux Mint 20.1 and at same time upgraded rednotebook to 2.22. It seems that the tag pane has vanished. Is there a way to make it visible.

**Reply by jendrikseipp** [reply]
The tag pane is deprecated, but you can make it visible by setting "showTagsPane=1" in ~/.rednotebook/configuration.cfg.

**Reply by pada-jb** [reply]
Got it, Thanks! 
Just an FYI on rednotebook usage. I use the tag window much more than the edit/preview window. Much of the data that I input is repetitive and it is much easier (for me) to add the tag and type 1-2 letters to find the input I am adding. Would be nice if that selected input would also show in the preview/edit space and not show up double in a search, but it is all good as long as i can view the tag pane. Thanks again and good work!

**Reply by pada-jb** [reply]
Does the same solution apply to Win 10? Set "showTagsPane=1" and no change to Rednotebook

**Reply by jendrikseipp** [reply]
Yes, that should work for all operating systems. The pane might be minimized, so try moving the horizontal slider to the left.

**Reply by pada-jb** [reply]
Got it. No slider. Resized the window and it appeared. Thanks again.

**Comment by VanceAyres** [comment]
I'm on a Debian derivative & was wondering if there was a location where I could download the DEB file of version 2.29.6 or newer so I can use shortcut keys again.
Thanks!

**Reply by jendrikseipp** [reply]
You can find deb files here: https://code.launchpad.net/~rednotebook/+archive/ubuntu/stable

**Reply by VanceAyres** [reply]
Thank you Jendrik.  I had seen those before but because the size was so small I discounted that they would work on Debian & thought somehow they would only work on Ubuntu.  It's nice to find a program that is so compact but yet so feature rich!!!
